src/applications/mhv/secure-messaging/sass/secure-messaging.scss

```diff
@@ -44,7 +44,8 @@ va-alert button {
 }
 
 va-modal va-radio {
-  margin-left: 12px;
+  margin-left: 0px;
+  margin-bottom: 24px;
 }
```

Comment:
me too, I would encourage to use DST classes. However, in this case I thought it would be more applicable since it is also addresses styling issues on Move modal as well